README.md

```diff
@@ -27,5 +27,82 @@ MODEL MANAGERS
 QUERY BUILDERS
 - QueryBuilder - Class to support creation of cypher queries to work with data in Neo4j  
 
-Dependencies:
+## A full example
+
+The example code below runs through a use case with an example of data loaders, model appliers, data providers, and model managers.
+
+#### Importing the data 
+
+The call to FileDataLoader in the code below connects to the database using environment variables for your host and log-in credentials. Alternatively you can use the format `FileDataLoader(host="bolt://...",credentials=("username","password"))` to connect. The call to clean_slate() empties the database, and then load_file reads in your data from the specified filepath.
+
+```python
+from model_managers import ModelManager
+from data_loaders import FileDataLoader
+from model_appliers import ModelApplier
+from data_providers import DataProvider
+
+fdl = FileDataLoader()
+fdl.clean_slate()
+fdl.load_file(
+    folder='examples/data/', 
+    filename='full_example_data.csv'
+)
+```
+
+Now the data will be populated within your database, but it wont be connected in a very meaningful way. We have \`Source Data Column\` nodes containing information about the columns of ur data - Study, Subject, Age and Sex, and \`Source Data Row\` nodes containing information about the two rows. The following code creates a model from the data using ModelManager. We are then setting some properties for classes Age and Subject via queries to the database.
+
+```python
+mm = ModelManager()
+mm.create_model_from_data()
+mm.query("MATCH (c:Class{label:'Subject'}) set c.short_label = 'USUBJID'")
+mm.query("MATCH (c:Class{label:'Age'}) set c.data_type = 'int'")
+
+```
+Here we can see we now have class and relationship nodes, illustrating connections between Study, Subject, Age and Sex. 
+
+![modelmanager example](examples/data/create_model_from_data%20example.PNG)
+
+Now using that class-relationship model we built using ModelManager, we can refactor our data and extract entities of the defined classes into separate nodes. In the code below we use `refactor_all` to do this, and we can see in the image that age values of 40 and 50 have been extracted into their own nodes. 
+
+```python
+ma = ModelApplier(mode="schema_CLASS")
+ma.refactor_all()
+
+```
+
+![modelapplier example](examples/data/create_model_from_data%20example.PNG)
```

Comment:
@shannonhaughton I think the hyperlink here is wrong - it shows the same picture as above